survey_goals_and_questions.md

```diff
@@ -0,0 +1,169 @@
+The draft survey itself.
+
+# Survey length
+Based on feedback from the community, a ~10 minute survey is ideal and would maximize participation likelihood.  Beyond length, we will make most questions optional so that participants can pick and choose questions if they are rushed or prefer not to answer.  Progress will be shown throughout.
+
+# Survey Topics
+Based on discussions with many in the community, the below topics were most frequently mentioned as items folks hoped to learn about the community / were perceived as most valuable to learn about the community.
+
+Each topic has:
+* Estimated timing
+* What we want to learn
+* Outcomes we hope to achieve by learning
+* List of questions to be asked
+
```

Comment:
can we ask as the first question something like, are you part of the MLOps community? yes/no

how do you engage with the community
- Slack
- podcast
- blog
- newsletter
- IRL meetups
- virtual events

and make it so you can choose more than one. the reason for this is if many people fill out the survey that arent in the community it may give us a biased view of who is actually in the community

Reply:
also asking these 3 questions would be really nice to know because we can learn what kind of stuff people are looking for when joining the community 

What are the top 3 priorities for your business / the company you work at
What are your top 3 personal priorities at work
What can this community do to help you / your biz achieve those goals